### DISCUSSION TITLE
melonDS on the Play Store

### DISCUSSION THREAD

**Discussion by RayyanAnsari** [rafaelvcaetano/melonDS-android#629] [Q&A]
Hi Rafael,
A user recently came to our forums asking about the Android version. This isn't the first time, but they said that they installed it through the Play Store. After taking a quick look it seems that this being an unofficial melonDS port isn't mentioned in the app description. Please could you state that it's unofficial, and maybe put a link to this  repository so users don't get muddled up?
Thanks.

**Comment by rafaelvcaetano** [comment]
Hey Rayyan! Thanks for letting me know. I usually go to the forums to see if anyone has been opening Android related issues, but I hadn't found any so I didn't imagine this being a problem. I'll try to make it clear in the app's description.
